scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -112,12 +122,12 @@ const GROUPS = [
                     lines: "Wireframe",
                 },
                 propertyID: "primitiveMode",
-            },
```

Comment:
I know this is fixing it according to the eslint rules, but I believe the trailing comma rule should be adjusted since it allows for less changes when adding or removing an element from the end of a list. feel free to ignore this comment, but I think it would be fine in this object to keep trailing comma's